html/pages/device/routing/bgp.inc.php

```diff
@@ -53,12 +53,12 @@
 
 echo ' | ';
 
-if ($vars['view'] == 'prefixes_vpnv4unicast') {
```

Comment:
How come you've changed the names of these?

Reply:
The short answer would probably be : Because that's the name of the graph files in html/includes/graphs/bgp :) 

But the reason behind it, is that this is built out of AFI and SAFI. VPNv4Unicast is a pure Cisco name, but the official structure is "ipv4 vpn" :

```
AFI=1 -> ipv4
AFI=2 -> ipv6
```
```
SAFI=1 -> unicast
SAFI=2 -> unicast
SAFI=128 -> vpn-unicast
SAFI=129 -> vpn-multicast
```
So combining them : 
```
AFI=1, SAFI=1, IPv4 unicast
AFI=1, SAFI=2, IPv4 multicast
AFI=1, SAFI=128, L3VPN IPv4 unicast -> cisco vpnv4 unicast
AFI=1, SAFI=129, L3VPN IPv4 multicast 

AFI=2, SAFI=1, IPv6 unicast
AFI=2, SAFI=2, IPv6 multicast
AFI=2, SAFI=128, L3VPN IPv6 unicast -> cisco vpnv6 unicast
AFI=2, SAFI=129, L3VPN IPv6 multicast 
```

Reply:
So the current LibreNMS master does not display anything for vpnv4 unicast, even if you have cisco routers. There are a few other inconsistancies in the BGP and VRF code, catching them as I build a new MPLS network in my company ;)